### DISCUSSION TITLE
All data type specifications in Mojo

### DISCUSSION THREAD

**Discussion by bsyedumar** [modular/modular#278] [General]
I have spent a fair amount of time trying to figure out all the natively supported type specification types in Mojo and I can't get beyond F64, SI, UI, and String after importing it. Would be really helpful to add a complete list of types in the docs.

**Comment by mojodojodev** [comment]
Have you seen [these docs](https://docs.modular.com/mojo/lib.html), and the [numeric types here](https://docs.modular.com/mojo/MojoStdlib/SIMD.html)?

**Reply by bsyedumar** [reply]
I saw the numeric types after the post, yes. Thanks!
